data/core/terrain-graphics.cfg

```diff
@@ -888,7 +888,7 @@ C*,K*,X*,Q*,W*,Ai,M*,*^V*,*^B*,_off^_usr#enddef
 {NEW:TRANSITION            (Ms,Ha)          (W*,S*)                     -171               hills/snow-to-water}
 {NEW:TRANSITION            (Ms,Ha)          (!,Ha,Qx*,Mm,Ms,Md)         -172               hills/snow}
 
-{NEW:TRANSITION            (Tb)             (!,M*,Tb)                   -184               forest/mushroom-base}
+{NEW:TRANSITION            (Tb,*^Uf*)       (!,M*,Tb)                   -184               forest/mushroom-base}
```

Comment:
`Tb` occurs in two places on this line, but `*^Uf*` only once - should it be like that?

Sorry for taking so long to start reviewing this, and I haven't started testing it in game yet.

Reply:
Yeah sure, it does not seem to do much but I have added it for consistency.

Reply:
You can test it in the editor by setting commenting hide=yes in terrain.cfg